.github/workflows/dev-ci-cd.yaml

```diff
@@ -87,31 +90,25 @@ jobs:
       # stop any existing docker compose that's running
       - name: Stop Docker Compose
         run: |
-          ssh submit-t3desk 'bash -s' < ${{ github.workspace }}/deploy/dev/dev-stop.sh
+          ssh submit06 'bash -s' < ${{ github.workspace }}/deploy/dev/dev-stop.sh
```

Comment:
Why the move to subMIT06? I'm not against it, just want to make sure that we're consciously changing to other machines.